tests/scripts/thread-cert/Cert_7_1_05_BorderRouterAsRouter.py

```diff
@@ -159,16 +159,17 @@ def verify(self, pv):
         # Step 5: The DUT MUST send a multicast MLE Data Response,
         #         including at least three Prefix TLVs (Prefix 1, Prefix2,
         #         and Prefix 3).
-        _dv_pkt = pkts.filter_wpan_src64(ROUTER).\
-            filter_LLANMA().\
-            filter_mle_cmd(MLE_DATA_RESPONSE).\
-            filter(lambda p: {
-                              Ipv6Addr(PREFIX_2001[:-3]),
-                              Ipv6Addr(PREFIX_2002[:-3]),
-                              Ipv6Addr(PREFIX_2003[:-3])
-                             } == set(p.thread_nwd.tlv.prefix)
-                   ).\
-            must_next()
+        with pkts.save_index():
```

Comment:
There might be CUR before DUT sends the MLE Data Response randomly delayed. However such CUR is just for keep-alive, should not have Address Registration TLV which has the the  addresses based on the three prefixes.

MED could only learn the three prefixes from the MLE Data Response sent by its parent, and sends CUR to register the new addresses afterwards. To fix the issue, Step6 below may find a CUR which registers the three prefix based addresses, and Step7 echos back these addresses. 

Thoughts?

Reply:
the step 6 and 7 checks are below the Step 7 comments for checking the address registration field in Step 7, added IID len check in Step6 for filtering out the no Address Registration TLV CUR in the second commit

Reply:
It may not be necessary to have `with pkts.save_index():` given the expecting CUR would always happen after this multicast MLE Data Response?

Reply:
This issue is caused by the packets index disordered in tiny time difference:
![image](https://user-images.githubusercontent.com/33887762/102717340-a0bfc600-431c-11eb-904b-664958ce483a.png)
MLE Data Response in index 75 and CUR in index 70, better to save the index here

Reply:
Thanks for the info - it is not disorder issue. It turns out to be that the DUT sends the MLE advertisement before the delayed MLE Data Response, and MED sends unicast Data Request to request new Network Data after learning the new version info from Leader Data TLV of MLE Advertisement, so CUR may happen before the multicast Data Response.

Fine to me to keep the `with pkts.save_index():`.